src/app/play/[contents_id]/page.js

```diff
@@ -0,0 +1,11 @@
+import PlayContainer from "src/components/Play/PlayContainer/PlayContainer.js";
+
+export default function Play(props) {
+  var contentId = props.params.contents_id;
```

Comment:
```suggestion
  const contentId = props.params.contents_id;
```
값이 변하지 않는 상수들은 `const`로 대체하는 것이 좋습니다!